### DISCUSSION TITLE
[Stundenplan] Raumänderung Lehrer

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#44] [User Stories]
### User Story 
Als Lehrer möchte ich die Räume ändern können, weil ich für manche Aufgaben / Stunden ein spezielles Equipment benötige z.B. den Computerraum
### Beschreibung:
Als Lehrer soll der Studenplan bearbeiten werden können. Dabei soll der Raum geändert und ein neuer gebucht werden können, falls dieser nicht beleget ist. Der Raum soll dann für alle anderen in diesem Zeitraum gesperrt werden.

### Akzeptanz-Kriterien:

- [ ] Lehrer kann den Raum wechseln (falls der gewünschte Raum nicht schon belegt ist) => Stundenplan wird aktualisiert
- [ ] Falls ein Lehrer den Raum für die Stunde beansprucht wird dieser für andere als belegt angezeigt

**Comment by Schmarvinius** [comment]
Das ist eher die Aufgabe eines Admins bzw die des Sekretariat und nicht des Lehrers
Die haben sich bei der Zusammenstellung des Plans ja was gedacht

**Reply by Kronprinz03** [reply]
Naja nicht wirklich als Lehrer will ich ja z.B. einfach in den Computerraum oder z.B. nicht in die Sporthalle dafür brauche ich nicht das Sekretariat
